feature-libs/pickup-in-store/jest.schematics.config.js

```diff
@@ -0,0 +1,34 @@
+const { pathsToModuleNameMapper } = require('ts-jest');
```

Comment:
maybe just copy the file from other libs? We made some changes in: https://github.com/SAP/spartacus/pull/16532